[CI] ReindexClientYamlTestSuiteIT test {yaml=reindex/10_basic/Response format for created} failing

**Build Scans:**
- [elasticsearch-periodic #12162 / openjdk21_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/2xqlklpsueuio)
- [elasticsearch-intake #38030 / part1](https://gradle-enterprise.elastic.co/s/3oidv2p22mwde)

**Reproduction Line:**
```
./gradlew ":modules:reindex:yamlRestTest" --tests "org.elasticsearch.index.reindex.ReindexClientYamlTestSuiteIT.test {yaml=reindex/10_basic/Response format for created}" -Dtests.seed=ED70CD724EC76873 -Dtests.locale=ha -Dtests.timezone=America/Moncton -Druntime.java=21
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.index.reindex.ReindexClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Byaml%3Dreindex%2F10_basic%2FResponse%20format%20for%20created%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: java.lang.AssertionError: field [indices.source.total.search.open_contexts] does not match the expected value
Expected: <0>
     but: was <1>
```

**Issue Reasons:**
- [main] 1 consecutive failures in test test {yaml=reindex/10_basic/Response format for created}
- [main] 1 consecutive failures in step openjdk21_checkpart1_java-matrix
- [main] 1 consecutive failures in pipeline elasticsearch-periodic
- [main] 2 failures in test test {yaml=reindex/10_basic/Response format for created} (0.4% fail rate in 469 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -208,9 +208,6 @@ tests:
 - class: org.elasticsearch.xpack.ml.integration.DatafeedJobsIT
   method: testStopLookback_forceTrue_closeJobFalse
   issue: https://github.com/elastic/elasticsearch/issues/146955
-- class: org.elasticsearch.index.reindex.ReindexClientYamlTestSuiteIT
-  method: test {yaml=reindex/10_basic/Response format for created}
-  issue: https://github.com/elastic/elasticsearch/issues/147005
 - class: org.elasticsearch.reindex.management.ReindexManagementClientYamlTestSuiteIT
   method: test {yaml=reindex/30_cancel_reindex/Cancel running reindex returns response and GET confirms completed}
   issue: https://github.com/elastic/elasticsearch/issues/146314

